__tests__/infrastructure/TransactionService.spec.js

```diff
@@ -15,7 +15,7 @@ describe('Transaction Service', () => {
 				}
 			};
 
-			const mockTransactionEffectiveFee = '0.001760';
+			const mockTransactionEffectiveFee = '0.001000';
```

Comment:
why this change? were tests failing?

Reply:
Because the `payloadSize` affected the fees.

https://github.com/symbol/explorer/blob/21cdac24417aea0bda23dce412ca1fefc2a22d54/__tests__/TestHelper.js#L28

Reply:
so, it was failing before?

Reply:
not, previously the payloadSize was set to `176`.

let me update some of the code.